packages/core/src/bodl/bodl-emitter-service.ts

```diff
@@ -212,10 +212,16 @@ export default class BodlEmitterService implements BodlService {
                 brand_name: item.brand,
                 currency: currencyCode,
                 category_names: item.categoryNames || [],
+                retail_price: item.retailPrice,
             };
         });
 
-        return [...customItems, ...physicalAndDigitalItems, ...giftCertificateItems];
+        return [...customItems, ...physicalAndDigitalItems, ...giftCertificateItems].map(
+            (item) => ({
+                ...item,
+                product_id: String(item.product_id),
```

Comment:
I cannot access the doc in the description.
This change makes the produce id as a string.
Is it intentional?

Reply:
I cannot access the doc in the description.
This change makes the product id as a string.
Is it intentional?